SimPEG/potential_fields/magnetics/receivers.py

```diff
@@ -51,6 +51,7 @@ def __init__(self, locations, components="tmi", **kwargs):
                     "by",
                     "bz",
                     "tmi",
+                    "tmi_exact",
```

Comment:
My thought that this would be nice to have implemented for the integral code as well down the road

Reply:
It would be, but I'd still recommend it to be an option of the `Simulation` as it's not actually a change in the thing the receiver records, but is a change in how the data is simulated.